.circleci/config.yml

```
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
+    - checkout
+
+    - run: |
+        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./architect
+        ./architect version
+    - run: ./architect build
+    - deploy:
```

What about add support JUnit reports? we have in other operator and it helps to give more info when something fails
```suggestion
    - store_test_results:
        path: /tmp/results
    - deploy:
```